.github/workflows/lock.yml

```
@@ -0,0 +1,22 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    # Run once a day
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+concurrency:
+  group: lock
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v4
+        with:
+          issue-inactive-days: '45'
```

```suggestion
          issue-inactive-days: 45
```

Will work as number and is actually documented as such in the parameters list.